.github/workflows/build-images.yml

```diff
@@ -0,0 +1,56 @@
+---
+name: Build Docker images for all OpenSlides services
+on: push
```

Comment:
better:
```yml
on:
    push:
        branches:
            - master
```
covers all code changes that happened on the given branch, either by a direct commit or merged pull request

Reply:
I like this. Done